src/styles/videoosd.scss

```diff
@@ -172,12 +172,6 @@
     margin-left: auto;
 }
 
-@keyframes spin {
```

Comment:
I would keep this one and remove the one below - less changes and the animation is defined before use.

Reply:
I see all `@keyframes` are defined at the end of the file.
I'll create two explicite spin animations, one for clockwise, one for counterclockwise, and use each animation in the right class.